crowdloans/projects/polkadot.json

```diff
@@ -160,7 +160,7 @@
     "overview": "Acala is the decentralized finance network and liquidity hub of Polkadot. It’s a layer-1 smart contract platform that’s scalable, Ethereum-compatible, and optimized for DeFi with built-in liquidity and ready-made financial applications.",
     "website": "https://acala.network/",
     "icon": "https://auction-us-east-2.s3.us-east-2.amazonaws.com/icons/chains/acala.svg",
-    "rewardStatus": "PARALLEL_IS_DISTRIBUTING",
+    "rewardStatus": "PARALLEL_TO_DISTRIBUTE",
```

Comment:
For Acala - it should now say "Parallel is distributing"
Production is not correct: https://crowdloan.parallel.fi/#/auction/rewards/polkadot

Reply:
Notion document is updated after I made the changes. Will add this and create a PR

Reply:
"Parallel is distributing"

Reply:
For Acala